### ISSUE TITLE
Passing arguments to the readthedoc build

### ISSUE THREAD

**Issue by iquah1** [readthedocs/readthedocs.org#9043]
## Details

[Docs Project URL](https://readthedocs.org/projects/openfhe-development) `https://readthedocs.org/projects/openfhe-development`
[Build URL](https://readthedocs.org/projects/openfhe-development/builds/16469588/) `https://readthedocs.org/projects/openfhe-development/builds/16469588/`
RTD username: ` iquah1`

## Expected Result

Passing arguments to the readthedocs build. If you look at the line:

`/home/docs/checkouts/readthedocs.org/user_builds/openfhe-development/envs/latest/bin/python -m sphinx -T -E -b html -d _build/doctrees -D language=en . _build/html`

You'll see a ton of lines being output to console. Most of those are errors and warnings. I'd like to pipe those out to an error file instead and if the build fails, the error file will be `cat`

## Actual Result

No such behavior is supported. Is there any way I can accomplish this?

**Comment by humitos** [comment]
Hi @iquah1! Unfortunately, we don't support piping that command to a file. I'm curious, what's the problem with the current behavior?

**Comment by iquah1** [comment]
The issue is that it spits out lots of logging messages which makes it difficult to parse on our end. I was hoping to just pipe them all to a file and only retrieve the logs if something goes wrong. If you click on the 4th cell from the bottom on my [RTD build](https://readthedocs.org/projects/openfhe-development/builds/16469588/) 

`latexmk -r latexmkrc -pdf -f -dvi- -ps- -jobname=openfhe-development -interaction=nonstopmode` 

you'll see that it spat out a ton of logs. The issue is really 2-fold:

1) it causes truncation which may mean we've lost important information (e.g in the case it errors out)
2) it makes it hard to parse the logs

___

Unfortunately, our method of handling local doc builds does not work (piping to a file). The warning logs are expected due to limitations of a library we are using

**Comment by humitos** [comment]
We don't support the use case you are describing, unfortunately. However, I'd suggest you subscribe to #9062 because we will allow running custom build commands in the future, where you will be able to do what you need.
